pages/pipelines/incoming_webhooks.md.erb

```diff
@@ -13,7 +13,26 @@ Secrets are pipeline-specific. A secret can’t prove that it hasn’t been acqu
 
 You can also provide the permitted IPs for GitHub Enterprise Server, BitBucket Server, GitLab Community, and GitLabEnterprise in your [organization's settings](https://buildkite.com/organizations/~/settings). To enable IP filtering for them, in the [Repository providers](https://buildkite.com/organizations/krnswry/repository-providers) tab, enter the allowed IP addresses. 
 
-For non-server BitBucket, IP filtering for webhooks is currently unavailable.
+For non-server BitBucket, IP filtering for webhooks is currently unavailable.  
+
+### Can the webhook that triggered a build be accessed in the UI or API?  
```

Comment:
```suggestion
### How do I access the webhook that triggered a build? 
```

Reply:
And the reword to the two options? 

With the first point, an example URL might help.